lib/report/generateReport.ts

```diff
@@ -273,6 +278,12 @@ export class CoverageView {
     }
   }
 
+  private async readMarkdown() {
+    const loader = new FileLoader({});
```

Comment:
can you add try catch here so that markdown loading failure doesn't block html report rendering?